spec/helpers/monitoring.js

```diff
@@ -268,7 +268,7 @@ function Monitoring() {
 
     this.selectGivenItem = function(item) {
         var itemTypeIcon = item.element(by.css('.type-icon'));
-        browser.actions().mouseMove(itemTypeIcon, {x: -50, y: -50}).mouseMove(itemTypeIcon).perform();
+        browser.actions().mouseMove(itemTypeIcon, {x: -45, y: -45}).mouseMove(itemTypeIcon).perform();
```

Comment:
@petrjasek @vied12  I was forced to reduce the value of coordinates passed to _mouseMove_. With the previous value, the test was failing as the browser click action was targeting a different element (highlight main menu instead of item checkbox)